google-calendar/operations.py

```diff
@@ -0,0 +1,178 @@
+"""
+Copyright start
+MIT License
+Copyright (c) 2024 Fortinet Inc
+Copyright end
+"""
+
+from requests import request
+from connectors.core.connector import get_logger, ConnectorError
+from .google_api_auth import *
+from .constants import *
+
+CALENDARS_API_VERSION = 'v3'
+
+logger = get_logger('google-calendar')
+
+
+def api_request(method, endpoint, connector_info, config, params=None, data=None, headers={}):
+    try:
+        go = GoogleAuth(config)
+        endpoint = go.host + "/" + endpoint
+        token = go.validate_token(config, connector_info)
+        headers['Authorization'] = token
+        headers['Content-Type'] = 'application/json'
+        response = request(method, endpoint, headers=headers, params=params, data=data, verify=go.verify_ssl)
+        if response.ok or response.status_code == 204:
```

Comment:
Add Curl Utils code here

# CURL UTILS CODE
            try:
                from connectors.debug_utils.curl_script import make_curl
                make_curl(method, endpoint, headers=headers, params=params, data=data, verify_ssl=self.verify_ssl)
            except Exception as err:
                logger.error(f"Error in curl utils: {str(err)}")